modules/rosapps/applications/cmdutils/uptime/uptime.c

```diff
@@ -24,7 +24,7 @@ int main(int argc, char* argv[])
         _tprintf(_T("  %.2u:%.2u  "), SystemTime.wHour, SystemTime.wMinute);
         _tprintf(_T("up %I64u days, %I64u:%I64u\n"), Days, Hours, Mins); /*%.2I64u secs*/
 #else
-        _tprintf(_T("System Up Time:\t\t%I64u days, %I64u Hours, %I64u Minutes, %.2I64u Seconds\n"),
+        _tprintf(_T("System Uptime:\t\t%I64u days, %I64u Hours, %I64u Minutes, %.2I64u Seconds\n"),
```

Comment:
I fear (same comment for the translation) that this change somehow breaks parsing:
https://en.wikipedia.org/wiki/Uptime#Using_systeminfo
The english version of the corresponding MS tool writes this as "System Up Time".

Reply:
Indeed, confirming that. So this string should be left unchanged for compatibility.